src/ncorr_commandLine.cpp

```diff
@@ -94,21 +83,10 @@ void exportStrainData(string outputFilesDirectory, string csvExport, strain_anal
 			}
 
 		}
-		if (exportVaribles[idx].compare("mean eyy") == 0) {
+		if (csvExport[idx].compare("mean eyy") == 0) {

```

Comment:
I would prefer to use an `else if` here. It doesn't change the behavior, I just think it makes it easier to read and understand.

Reply:
Yeah, this is ugly garbage that will get refactored fairly hard at some point. I just said screw it, get her done tonight.